rpm/expat.spec

```diff
@@ -35,40 +34,41 @@ The expat-devel package contains the libraries, include files and documentation
 to develop XML applications with expat.
 
 %prep
-%setup -q -n expat-%{version}/upstream/expat
+%autosetup -p1 -n %{name}-%{version}/upstream
+
+%build
+pushd expat
```

Comment:
using 'pushd' without 'popd' - why not just 'cd'?

Reply:
Guess pushd out of a habit on .spec files, popd just skipped as there was no reason to get back.

Reply:
To me at least it looks incomplete, like something is missing or got accidentally deleted.

[I've sort of assumed pushd would be a bashism i.e. something to avoid. But I've seen them elsewhere too, so maybe I'm wrong about it.]

Reply:
It's sometimes nice when the builds need to do things in different paths but on this case sure less need. Can just use cd instead, all the same to me.

Reply:
Done.

Reply:
> [I've sort of assumed pushd would be a bashism i.e. something to avoid. But I've seen them elsewhere too, so maybe I'm wrong about it.]

Just to comment this: Yes, it is bash-ism but we only ever use bash to build packages so those can be used there. They can't exist in scriptlets but usually there is no need for that anyway.